query-engine/driver-adapters/src/conversion/postgres.rs

```diff
@@ -0,0 +1,55 @@
+use crate::conversion::JSArg;
+use chrono::format::StrftimeItems;
+use once_cell::sync::Lazy;
+use serde_json::value::Value as JsonValue;
+
+static TIME_FMT: Lazy<StrftimeItems> = Lazy::new(|| StrftimeItems::new("%H:%M:%S%.f"));
+
+pub fn values_to_js_args(values: &[quaint::Value<'_>]) -> serde_json::Result<Vec<JSArg>> {
```

Comment:
For the moment the rest of flavors follow the previous generic `value_to_js_args` function in the conversion.rs module. At some point that will be removed, but NOT dryed up. Repetition is better than the wrong abstraction

Reply:
Can you add a comment where this code exists, to make it explicit it is duplicated intentionally for now?